### DISCUSSION TITLE
How to use in-line eval-rst to cross-reference Python object

### DISCUSSION THREAD

**Discussion by joe-agent** [executablebooks/MyST-Parser#622] [Q&A] [answered]
Previously, I used the syntax below in `rst` to cross-reference Python class auto-generated by autodoc like below:

`1. Add a new microservice class in ``service.py`` inheriting from :py:func:`abc.servicebus.microservice.microservice``

Now after I convert the `rst` to `myst`, I have to make a line break to make `{eval-rst}` to work:
````
    1. Add a new microservice class in ``service.py`` inheriting from 
       ```{eval-rst}
        :py:class:`abc.servicebus.microservice.microservice`
       ```
````

How do I use `{eval-rst}` inline without line break in a markdown file?
Can anyone help? @choldgraf, @chrisjsewell

**Comment by chrisjsewell** [comment, marked as answer]
Heya, @joe-agent you should generally never need `{eval-rst}`, you can access all roles and directives in MyST, e.g. `` {py:class}`abc.servicebus.microservice.microservice` ``

`{eval-rst}` is only for the rare occassions, when something has some "hard-coding" of rST
